R/zzz.R

```diff
@@ -0,0 +1,4 @@
+.onAttach = function(...) { #nolint
+  if (!interactive()) return()
+  packageStartupMessage("See https://www.sonatype.com/ for details.")
```

Comment:
Hah, sadly I don't think anyone will get a ton of value being sent here. When would this link appear?

Reply:
This would appear when someone calls: